### PR TITLE
Update/rename toolbelt

### PR DESCRIPTION
**Pull request by carolinamenezes**
#### Types of changes

- [x] Content fix
- [ ] Answer sheet fix
- [ ] Course refactor
- [ ] New course :rocket:
- [ ] Script bug fix
- [ ] Script feature

### CODE REVIEW

File: courses/basic-blocks/steps/00_setup/pt.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Antes de colocar as mãos na massa e aprender mais sobre o Store Framework no VTEX IO, você precisará definir algumas configurações básicas, como:
 
 - Instalação do **Git**;
-- Instalação do **Toolbelt**;
+- Instalação da **CLI da VTEX**;
```
Comment (htadashi) [inline courses/basic-blocks/steps/00_setup/pt.md:8]: Não seria CLI do VTEX **IO**? (Para acompanhar a versão em inglês)

File: courses/basic-blocks/steps/00_setup/en.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 Before getting your hands dirty and learning more about VTEX IO's Store Framework, you'll need to set up a few basic configurations, such as:
 
 - Installing **Git**;
-- Installing **Toolbelt**;
+- Installing **VTEX IO's CLI**;
```
Comment (htadashi) [inline courses/basic-blocks/steps/00_setup/en.md:8]: The GitHub repository of [VTEX IO CLI](https://github.com/vtex/toolbelt) is still named toolbelt, so maybe it is a good idea to change (only this line) to "Installing VTEX IO's CLI (former VTEX toolbelt)" or something similar?

Comment (carolinamenezes) [reply courses/basic-blocks/steps/00_setup/en.md:8]: Toolbelt is the name of the GitHub project of the VTEX IO CLI, but the external developer doesn't interact with the GitHub project and doesn't even need to know about it. The developer/documentation user is familiar with the concept of a CLI, so using "Toolbelt" just makes things more complicated.

File: courses/basic-blocks/steps/00_setup/pt.md (hunk 1 above)
Comment (carolinamenezes) [reply courses/basic-blocks/steps/00_setup/pt.md:8]: Verdade! Fiz correndo

Comment (carolinamenezes) [reply courses/basic-blocks/steps/00_setup/pt.md:8]:
```suggestion
- Instalação da **CLI do VTEX IO**;
```

File: courses/basic-blocks/steps/00_setup/pt.md [hunk 3]
```diff
@@ -18,16 +18,16 @@ Instale o Git em seu computador clicando no link abaixo e selecionando seu siste
 
 [https://git-scm.com/downloads](https://git-scm.com/downloads)
 
-## Instalando o Toolbelt
+## Instalando a CLI da VTEX
```
Comment (carolinamenezes) [inline courses/basic-blocks/steps/00_setup/pt.md:21]:
```suggestion
## Instalando a CLI do VTEX IO
```

File: courses/basic-blocks/steps/00_setup/pt.md [hunk 4]
```diff
@@ -18,16 +18,16 @@ Instale o Git em seu computador clicando no link abaixo e selecionando seu siste
...
 
-**Toolbelt** é uma ferramenta VTEX **de linha de comando**. Ele permite que você execute qualquer atividade na plataforma, como criar um novo _workspace_ de desenvolvimento, fazer login em uma conta VTEX, desenvolver novas aplicações ou gerenciar os já existentes, etc.
+**A CLI da VTEX** é uma ferramenta VTEX **de linha de comando**. Ele permite que você execute qualquer atividade na plataforma, como criar um novo _workspace_ de desenvolvimento, fazer login em uma conta VTEX, desenvolver novas aplicações ou gerenciar os já existentes, etc.
```
Comment (carolinamenezes) [inline courses/basic-blocks/steps/00_setup/pt.md:23]:
```suggestion
**A CLI do VTEX IO** é uma ferramenta VTEX **de linha de comando**. Ele permite que você execute qualquer atividade na plataforma, como criar um novo _workspace_ de desenvolvimento, fazer login em uma conta VTEX, desenvolver novas aplicações ou gerenciar os já existentes, etc.
```

File: courses/basic-blocks/steps/00_setup/pt.md [hunk 5]
```diff
@@ -18,16 +18,16 @@ Instale o Git em seu computador clicando no link abaixo e selecionando seu siste
...
 
-Uma vez que é o Toolbelt que estabelece a comunicação entre o desenvolvedor e a plataforma, você precisará dele para realizar todas as atividades propostas durante esse e os outros cursos.
+Uma vez que é a CLI da VTEX que estabelece a comunicação entre o desenvolvedor e a plataforma, você precisará dele para realizar todas as atividades propostas durante esse e os outros cursos.
```
Comment (carolinamenezes) [inline courses/basic-blocks/steps/00_setup/pt.md:25]:
```suggestion
Uma vez que é a CLI do VTEX IO que estabelece a comunicação entre o desenvolvedor e a plataforma, você precisará dele para realizar todas as atividades propostas durante esse e os outros cursos.
```

File: courses/basic-blocks/steps/00_setup/pt.md [hunk 6]
```diff
@@ -18,16 +18,16 @@ Instale o Git em seu computador clicando no link abaixo e selecionando seu siste
...
 
 1. Instale o [**Node.js**](https://nodejs.org/). Se você estiver usando um MAC, instale também [**Yarn**](https://yarnpkg.com/);
 2. Execute `npm i -g vtex` em seu terminal se estiver usando Windows ou `yarn global add vtex` se estiver usando MAC;
 
-Você pode executar `vtex` para confirmar se a instalação do Toolbelt foi conforme o esperado.
+Você pode executar `vtex` para confirmar se a instalação da CLI da VTEX ocorreu conforme o esperado.
```
Comment (carolinamenezes) [inline courses/basic-blocks/steps/00_setup/pt.md:30]:
```suggestion
Você pode executar `vtex` para confirmar se a instalação da CLI do VTEX IO ocorreu conforme o esperado.
```

File: courses/content-workflow/overview/pt.md [hunk 7]
```diff
@@ -1,3 +1,3 @@
-## Utilizando o Site Editor e o VTEX Toolbelt para gerenciar e customizar sua loja
+## Utilizando o Site Editor e a CLI da VTEX para gerenciar e customizar sua loja
```
Comment (carolinamenezes) [inline courses/content-workflow/overview/pt.md:1]:
```suggestion
## Utilizando o Site Editor e a CLI do VTEX IO para gerenciar e customizar sua loja
```

File: courses/content-workflow/overview/pt.md [hunk 8]
```diff
@@ -1,3 +1,3 @@
...
 
-Este curso tem como objetivo introduzir os desenvolvedores em como utilizar o Site Editor e o VTEX Toolbelt, de forma a aprender como gerenciar os diferentes tipos de workspace, publicar e instalar aplicações e, por fim, como customizar sua frente de loja.
+Ao final deste curso, você irá aprender a gerenciar os diferentes tipos de workspace, publicar e instalar aplicações, e customizar a frente de loja utilizando o **Site Editor** e a **CLI da VTEX**.
```
Comment (carolinamenezes) [inline courses/content-workflow/overview/pt.md:3]:
```suggestion
Ao final deste curso, você irá aprender a gerenciar os diferentes tipos de workspace, publicar e instalar aplicações, e customizar a frente de loja utilizando o **Site Editor** e a **CLI do VTEX IO**.
```

File: courses/content-workflow/steps/02_publishing-app/pt.md [hunk 9]
```diff
@@ -2,11 +2,11 @@
 
 ## Introdução
 
-Após ser apresentado ao conceitos dos diferentes tipos de _workspaces_, vamos então ensinar como publicar uma aplicação no VTEX IO. Para realizar operações como esta, são utilizados comandos no [VTEX Toolbelt](https://developers.vtex.com/docs/vtex-io-documentation-toolbelt), nossa CLI que dá acesso aos recursos da plataforma.
+Após ser apresentado ao conceitos dos diferentes tipos de _workspaces_, vamos então ensinar como publicar uma aplicação no VTEX IO. Para realizar operações como esta, são utilizados comandos na [CLI da VTEX](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-vtex-io-cli-command-reference), nossa CLI que dá acesso aos recursos da plataforma.
 
-## Usando o VTEX Toolbelt para publicar uma _app_
+## Usando a CLI da VTEX para publicar uma _app_
```
Comment (carolinamenezes) [inline courses/content-workflow/steps/02_publishing-app/pt.md:7]:
```suggestion
## Usando a CLI do VTEX IO para publicar uma _app_
```

File: courses/basic-blocks/steps/00_setup/en.md (hunk 2 above)
Comment (htadashi) [reply courses/basic-blocks/steps/00_setup/en.md:8]: Got it, makes sense!

**Review by htadashi — APPROVED** [review]
LGTM

File: courses/basic-blocks/steps/00_setup/en.md (hunk 2 above)
Comment (carolinamenezes) [reply courses/basic-blocks/steps/00_setup/en.md:8]: Can you approve the PR so I can merge it? 🙊🙈

**Pull request merged**